src/neuroconv/schemas/base_metadata_schema.json

```
@@ -0,0 +1,171 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "base_metafile.schema.json",
+  "title": "Base schema for the metafile",
+  "description": "Base schema for the metafile",
+  "version": "0.1.0",
+  "type": "object",
+  "required": ["NWBFile"],
+  "properties": {
+    "NWBFile": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "keywords": {
+          "description": "Terms to search over",
+          "type": "array",
+          "items": {
+            "title": "keyword",
+            "type": "string"
+          }
+        },
+        "experiment_description": {
+          "type": "string",
+          "description": "general description of the experiment"
```

It's relatively minor, but...

Stylistically there are divergences in the descriptions for these fields - some are just a bunch of lower case text with no punctuation, some are full proper sentences.

I'd recommend we standardize these to use full proper sentence structure; a part of the eventual plan for the GUIDE could include having a little 'info' hover button or something next to each field which describes what it means, and I'm fairly sure the JSON field descriptions are the way that info would be communicated

---

```suggestion
          "description": "General description of the experiment."
```

 Here is the description for the NWB Schema. I think you are right that they should be standardized and hold meaningful details, but I think we should ideally do it at that level.